include/util.py

```diff
@@ -0,0 +1,36 @@
+import re
+import datetime
+
+try:
+    import ujson as json
+except ImportError:
+    import json  # noqa
+
+
+try:
+    import ciso8601
+except ImportError:
+    ciso8601 = None
+
+try:
+    STR_TYPE = basestring
+except NameError:
+    STR_TYPE = str
+
+
+def parse_datetime(s):
+    if not s:
+        return s
+
+    if ciso8601:
+        return ciso8601.parse_datetime(s)
+
+    # strptime can't parse +00:00
```

Comment:
Would it make sense to fall back on `django.utils.dateparse.parse_datetime`? It supports timezone offsets.